web/gui/dashboard/src/utils/createRequestAction.ts

```diff
@@ -0,0 +1,24 @@
+import { createAction } from "redux-act"
+
+// slightly simplified version of the creator used in the cloud
+// we will unify it when some typing issues will be fixed (cloud version didn't warn on bad payload)
```

Comment:
I believe, now it has been improved in Cloud, so you might want to take a look. Also there are some convenient features like error-handling saga.